### DISCUSSION TITLE
Telegram Channel backup

### DISCUSSION THREAD

**Discussion by FiorenMas** [FiorenMas/Revanced-And-Revanced-Extended-Non-Root#32] [Announcements]
I have created a backup Telegram channel in case this repository is taken down by a DMCA claim. If you don't want to lose contact with this project, join this channel. I will announce the new repository there.
https://t.me/fiorenmas

**Comment by quyleanh** [comment]
If you are scared, go home :v

**Reply by FiorenMas** [reply]
Already in home -_-
